controls/stig_rhel9.yml

```diff
@@ -1866,16 +1866,8 @@ controls:
             RHEL 9 must implement DOD-approved encryption ciphers to protect the confidentiality
             of SSH client connections.
         rules:
-            - harden_sshd_ciphers_openssh_conf_crypto_policy
-            - sshd_approved_ciphers=stig_rhel9
+            - sshd_include_crypto_policy
         status: automated
-    -   id: RHEL-09-255064
```

Comment:
RHEL-09-255064 still seems to exist

See https://stigaview.com/products/rhel9/latest/RHEL-09-255064/